schwifty/bank_registry/manual_es.json

```diff
@@ -1,130 +1,1546 @@
+      "primary": true,
+      "short_name": "CAJA DE AHORROS DE SALAMANCA Y SORIA"
+  },
+  {
+      "bank_code": "0198",
+      "bic": "BCOEESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO COOPERATIVO",
+      "primary": true,
+      "short_name": "BANCO COOPERATIVO"
+  },
+  {
+      "bank_code": "0003",
+      "bic": "BDEPESM1XXX",
+      "country_code": "ES",
+      "name": "BANCO DE DEPOSITOS, S.A.",
+      "primary": true,
+      "short_name": "BANCO DE DEPOSITOS"
+  },
+  {
+      "bank_code": "0169",
+      "bic": "NACNESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO DE LA NACION ARGENTINA",
+      "primary": true,
+      "short_name": "BANCO DE LA NACION ARGENTINA"
+  },
+  {
+      "bank_code": "0059",
+      "bic": "MADRESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO DE MADRID,S.A.",
+      "primary": true,
+      "short_name": "BANCO DE MADRID"
+  },
+  {
+      "bank_code": "0155",
+      "bic": "BRASESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO DO BRASIL AG - ESPANA",
+      "primary": true,
+      "short_name": "BANCO DO BRASIL"
+  },
+  {
+      "bank_code": "0131",
+      "bic": "BESMESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO ESPIRITO SANTO S A SUCUR",
+      "primary": true,
+      "short_name": "NOVO BANCO"
+  },
+  {
+      "bank_code": "0184",
+      "bic": "BEDFESM1XXX",
+      "country_code": "ES",
+      "name": "BANCO EUROPEO DE FINANZAS",
+      "primary": true,
+      "short_name": "BANCO EUROPEO DE FINANZAS"
+  },
+  {
+      "bank_code": "0220",
+      "bic": "FIOFESM1XXX",
+      "country_code": "ES",
+      "name": "BANCO FINANTIA SOFINLOC S.A.",
+      "primary": true,
+      "short_name": "BANCO FINANTIA"
+  },
+  {
+      "bank_code": "0232",
+      "bic": "INVLESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO INVERSIS NET S.A.",
+      "primary": true,
+      "short_name": "BANCO INVERSIS"
+  },
+  {
+      "bank_code": "0487",
+      "bic": "GBMNESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO MARE NOSTRUM",
+      "primary": true,
+      "short_name": "BANCO MARE NOSTRUM"
+  },
+  {
+      "bank_code": "0186",
+      "bic": "BFIVESBBXXX",
+      "country_code": "ES",
+      "name": "BANCO MEDIOLANUM, S.A.",
+      "primary": true,
+      "short_name": "BANCO MEDIOLANUM"
+  },
+  {
+      "bank_code": "0238",
+      "bic": "PSTRESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO PASTOR",
+      "primary": true,
+      "short_name": "BANCO PASTOR"
+  },
+  {
+      "bank_code": "0072",
+      "bic": "PSTRESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO PASTOR",
+      "primary": true,
+      "short_name": "BANCO PASTOR"
+  },
+  {
+      "bank_code": "0235",
+      "bic": "PICHESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO PICHINCHA ESPANA",
+      "primary": true,
+      "short_name": "BANCO PICHINCHA"
+  },
+  {
+      "bank_code": "0004",
```

Comment:
There are several entries for Banco Popular with different domestic bank codes but the same BIC. Do all of those bank codes refer to the same bank branch or are those different branches? Could that maybe reflected in the `"name"` of the bank?